src/nxdoc/nuxeo-server/runtime-and-component-model/runtime-and-component-model-tutorials/how-to-define-a-runtime-xmap-object.md

```diff
@@ -90,6 +95,54 @@ Here is a reference of supported built-in types:
 
 Note that primitive types can also be used (boolean instead of Boolean for instance) but a default value will automatically be resolved in that case, and it will not be possible to check whether the XML held the corresponding mapping or not (and this can be important for contribution merging logics).
 
+The `@Node` also accepts additional configuration to define a fallback path for a field retrieval:
```

Comment:
Explain "fallback path". Suggestion: `fallback path (another element or attribute in the XML)`

Reply:
I also fixed the invalid sample (missing a comma)